code/asteroid/asteroid.cpp

```diff
@@ -2133,7 +2145,7 @@ void asteroid_frame()
 		return;
 	}
 
-	if (Default_asteroid_throwing_behavior) {
+	if (Asteroid_field.target_names.empty()) {
```

Comment:
Removing `Default_asteroid_throwing_behavior` is fine, but throw in a little comment here clarifying that this is being done because we're falling back to default behavior.